sdk/instrumentation/instrumentation-opentelemetry/review/instrumentation-opentelemetry.api.md

```diff
@@ -0,0 +1,71 @@
+## API Report File for "@azure/instrumentation-opentelemetry"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AzureLogger } from '@azure/logger';
+import type * as coreTracing from '@azure/core-tracing';
+import { InstrumentationBase } from '@opentelemetry/instrumentation';
+import { InstrumentationConfig } from '@opentelemetry/instrumentation';
+import { InstrumentationModuleDefinition } from '@opentelemetry/instrumentation';
+import { Instrumenter } from '@azure/core-tracing';
+import { InstrumenterSpanOptions } from '@azure/core-tracing';
+import { Span } from '@opentelemetry/api';
+import { SpanContext } from '@opentelemetry/api';
+import { SpanStatus } from '@azure/core-tracing';
+import { TracingContext } from '@azure/core-tracing';
+import { TracingSpan } from '@azure/core-tracing';
+import { TracingSpanContext } from '@azure/core-tracing';
+
+// @public
+export class AzureSDKInstrumentation extends InstrumentationBase {
```

Comment:
any chance we can get away with not having this be a class and also not doing cross-package inheritance? The latter has burned us a few times and the former is more because I resist classes whenever we're not creating a service client. 😄

Reply:
Belatedly realized this was coming from OT, agreed we might not be able to avoid it

Reply:
Ended up using Jeff's suggestion and this is now completely hidden from the public API using `createAzureInstrumentation` instead...